test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatAddPkgTests.cs

```diff
@@ -745,6 +746,117 @@ public async Task AddPkg_ConditionalAddWithUserInputFramework_Success(string pac
             }
         }
 
+        [Fact]
+        public async Task AddPkg_WithAdditionalSourceFeed_NoVersionSpecified_Success()
```

Comment:
Tests are still failing. 

Please ping me when they are passing.

Reply:
All tests are passing. 😄

Reply:
Unfortunately I pushed another small change right after it, so have to wait that one to clear.